sway/input_state.c

```diff
@@ -48,7 +48,7 @@ void release_key(keycode key) {
 	}
 }
 
-struct pointer_state pointer_state = {0, 0, {0, 0}, {0, 0, 0}, {0, 0, 0, 0}};
+struct pointer_state pointer_state = {0, 0, {0, 0}, {0, 0, {0, 0}}, {0, 0, 0, 0, 0, 0, 0, 0}};
```

Comment:
Could just remove the initializer since global and static variables are initialized to zero by default. This thing looks ugly.